src/test/java/com/commercetools/sync/commons/exceptions/InvalidAttributeDefinitionExceptionTest.java

```diff
@@ -0,0 +1,21 @@
+package com.commercetools.sync.commons.exceptions;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class InvalidAttributeDefinitionExceptionTest {
+
+    @Test
+    void invalidAttributeDefinitionException_WithMessageAndNoCauses_ShouldBuildExceptionCorrectly() {
```

Comment:
We have discussed this already: someone could change the inner constructor logic by mistake (which is a bit low probablity but still) then this test would fail, which is good. Also for coverage 😉